packages/client/src/links/internals/httpUtils.ts

```diff
@@ -138,6 +138,7 @@ export const jsonHttpRequester: Requester = (opts) => {
 export type HTTPRequestOptions = HTTPBaseRequestOptions &
   ContentOptions & {
     headers: () => HTTPHeaders | Promise<HTTPHeaders>;
+    TextDecoder?: TextDecoder;
```

Comment:
We may use `TextDecoderEsque` here as well, just for the consistency's sake.

Reply:
good catch